src/main/java/de/tum/cit/aet/artemis/core/dto/CourseExistingExerciseDetails.java

```diff
@@ -0,0 +1,7 @@
+package de.tum.cit.aet.artemis.core.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record CourseExistingExerciseDetails(String[] exerciseTitles, String[] shortNames) {
```

Comment:
```suggestion
public record CourseExistingExerciseDetails(Set<String> exerciseTitles, Set<String> shortNames) {
```
Otherwise other developers might think that the title at index i corresponds to the shortName at Index i